standard-dev.cfg

```diff
@@ -0,0 +1,45 @@
+# Standard development config to be used in standardized policies.
+# May or may not be used by more sophisticated policies.
+
+# Usage:
+# [buildout]
+# extends =
+#     test-policy.cfg
+#     https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/plone-development.cfg
+#     https://raw.githubusercontent.com/4teamwork/gever-buildouts/master/standard-dev.cfg
+#
+# ogds-db-name = ???
+
+[buildout]
+extends =
+    http://buildout-proxy.4teamwork.ch/4teamwork/opengever-buildouts/master/ruby-gems.cfg
+
+instance-eggs +=
+    opengever.core
+    psycopg2
+
+zcml-additional-fragments +=
+    <configure
+        xmlns="http://namespaces.zope.org/zope"
+        xmlns:db="http://namespaces.zope.org/db">
+
+        <include package="z3c.saconfig" file="meta.zcml" />
+
+        <db:engine name="opengever.db"
+          url="postgresql+psycopg2:///${buildout:ogds-db-name}" pool_recycle="3600" />
+        <db:session name="opengever" engine="opengever.db" />
+    </configure>
+
+
+[instance]
+zserver-threads = 4
```

Comment:
I just moved that from the policy. The 4 thread configuration comes from the template: https://github.com/4teamwork/opengever.core/blob/master/opengever/policytemplates/policy_template/opengever.+package.name+/development.cfg.bob#L12

I agree that 4 threads are too much for local development.
I also think we can easily switch to 1 thread: if we run into problems we can safely increase it since its only development.

Should I change it here to `1` then?